projects/1.20/assets/valhelsia-structures/valhelsia_structures/lang/zh_cn.json

```
@@ -0,0 +1,206 @@
+{
+    "itemGroup.valhelsia_structures": "英灵大陆|建筑结构",

```

```suggestion
    "itemGroup.valhelsia_structures": "英灵大陆丨建筑结构",
```
创造模式物品栏的分割符应当使用汉字`丨`（gun3），有特殊需求的除外。
详见https://cfpa.site/TransRules/ 的 **4. 专有名词 - 模组名** 一节。
此类问题请在您的翻译文件中全文通查并修改。